src/Styleguide/Utils/HorizontalPadding.tsx

```diff
@@ -0,0 +1,26 @@
+import { space, SpacingUnit } from "@artsy/palette"
+import styled from "styled-components"
+import { media } from "../../Components/Helpers"
+
+interface PaddingProps {
```

Comment:
Not _strictly_ necessary but good to note that with TypeScript's `--emitDeclaration` prop we've begun establishing a convention around naming that follows 

```
// MyFileName.tsx

export interface MyFileNameProps {} 

export class / function / whatever MyFileName<MyFileNameProps>
```

Any given file can have multiple interfaces so it doesn't work 100%, but for simple combinations this pattern seems to work well. It also helps avoid naming clashes.